src/gui/generalsettings.ui

```diff
@@ -48,6 +48,53 @@
     </widget>
    </item>
    <item row="2" column="0">
+    <widget class="QGroupBox" name="advancedGroupBox">
```

Comment:
Hm. By our design philosohpy, we shall avoid settings on the client side. The client should be smart enough to decide when to use zsync.
(And it anyway needs server support to work at all... or maybe not because in the zsync philosophy the server can be dumb and one could just provide a .zsync file additionally)

Reply:
I was asked to add that by @hodyroff, so that we could merge while testing. See: https://github.com/owncloud/core/issues/16162#issuecomment-337823287

Reply:
During testing I would recommend that it's still easily disabled by end-users and this provides that, again as requested by @hodyroff, See: https://github.com/owncloud/core/issues/16162#issuecomment-337823287

Reply:
Well, the problem is that this general settings page is already way too big, so adding more field in this tab is not a good idea at all.